commons/src/store/contact-avatar.ts

```diff
@@ -0,0 +1,30 @@
+import { writable } from "svelte/store";
+import { fetchContactImage } from "@commons";
+import type { ContactsQuery } from "@commons/types/Contacts";
+
+const contactAvatarMap: Record<string, string> = {};
+
+function initializeContactAvatars() {
+  const { subscribe, set } = writable<Record<string, string>>({});
+  return {
+    subscribe,
+    getContactAvatar: async (
+      query: ContactsQuery,
+      contact_id: string,
+      refresh = false,
+    ) => {
+      if (!contactAvatarMap[contact_id] || refresh) {
+        const avatar = await fetchContactImage(query, contact_id)
+          .then((res) => res)
+          .catch(() => "");
+        if (avatar) {
+          contactAvatarMap[contact_id] = avatar;
+        }
+      }
+      return contactAvatarMap[contact_id];
```

Comment:
Since `contactAvatarMap[contact_id]` is set to the value of `avatar`, why not make this 
```
contactAvatarMap[contact_id] = await fetchContactImage(query, contact_id)
  .then((res) => res)
  .catch(() => "");
  return contactAvatarMap[contact_id];
```